resources/metadata/kanikoExecute.yaml

```
@@ -36,7 +36,7 @@ spec:
           - --skip-tls-verify-pull
           # fixing Kaniko issue https://github.com/GoogleContainerTools/kaniko/issues/1586
           # as per comment https://github.com/GoogleContainerTools/kaniko/issues/1586#issuecomment-945718536
-          - --ignore-path=/busybox
+          - --ignore-path=/workspace
```

```suggestion
          - --ignore-path=/workspace
          - --ignore-path=/busybox
```

As per https://github.com/GoogleContainerTools/kaniko/issues/1586#issuecomment-945718536 we might want to still ignore `/busybox` folder

---

> As per [GoogleContainerTools/kaniko#1586 (comment)](https://github.com/GoogleContainerTools/kaniko/issues/1586#issuecomment-945718536) we might still want to ignore `/busybox` folder

You are probably right. `--cleanup` is not used though and if someone adds `--cleanup`, the defaults wont't apply. But it cannot hurt anyway.